Tests/SwiftParserTest/translated/UnclosedStringInterpolationTests.swift

```diff
@@ -126,7 +126,8 @@ final class UnclosedStringInterpolationTests: XCTestCase {
       fixedSource: ##"""
         _ = """
         \(
-        """""")"""
+        """ """)
+        """
```

Comment:
Here we require a whitespace because it's a multiline literal. 
A whitespace is a newline or a space.